### DISCUSSION TITLE
Search for question results

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#136] [Ideas]
The only way to see the question results is by exporting the orders or attendees. This is great but cumbersome if you regularly need to search for various details.

It would be amazing to be able to search question results within each module (attendee or order) and/or add some of the questions as a column to relevant result table in orders or attendees.

**Comment by daveearley** [comment]
This would be useful alright. I'll add it to the roadmap.

**Reply by justynpride** [reply]
Thanks. Glad the thought was helpful.
